soh/soh/Enhancements/randomizer/location_list.cpp

```diff
@@ -1279,6 +1312,40 @@ void Rando::StaticData::InitLocationTable() {
 
+    //Beehives    Randomizer Check                                            Randomizer Check                           Quest         Type            Area                         Actor ID        Scene ID             Params                        Flags Short Name                             Spoiler Name                              Hint Text Key                               Vanilla Item   Categories            Spoiler Collection Check                     Collection Check Group                             Vanilla Progression
+    locationTable[RC_KF_STORMS_GROTTO_BEEHIVE_LEFT]          = Location::Base(RC_KF_STORMS_GROTTO_BEEHIVE_LEFT,          RCQUEST_BOTH, RCTYPE_BEEHIVE, RCAREA_KOKIRI_FOREST,         ACTOR_OBJ_COMB, SCENE_GROTTOS,      TWO_ACTOR_PARAMS(-144, 0x2C), 0x00, "Storms Grotto Beehive Left",          "KF Storms Grotto Beehive Left",          RHT_KF_STORMS_GROTTO_BEEHIVE_LEFT,          RG_BLUE_RUPEE, {Category::cBeehive}, SpoilerCollectionCheck::RandomizerInf(0, 0), SpoilerCollectionCheckGroup::GROUP_KOKIRI_FOREST,  false);
+    locationTable[RC_KF_STORMS_GROTTO_BEEHIVE_RIGHT]         = Location::Base(RC_KF_STORMS_GROTTO_BEEHIVE_RIGHT,         RCQUEST_BOTH, RCTYPE_BEEHIVE, RCAREA_KOKIRI_FOREST,         ACTOR_OBJ_COMB, SCENE_GROTTOS,      TWO_ACTOR_PARAMS(121,  0x2C), 0x00, "Storms Grotto Beehive Right",         "KF Storms Grotto Beehive Right",         RHT_KF_STORMS_GROTTO_BEEHIVE_RIGHT,         RG_RED_RUPEE,  {Category::cBeehive}, SpoilerCollectionCheck::RandomizerInf(0, 0), SpoilerCollectionCheckGroup::GROUP_KOKIRI_FOREST,  false);
+    locationTable[RC_LW_NEAR_SHORTCUTS_GROTTO_BEEHIVE_LEFT]  = Location::Base(RC_LW_NEAR_SHORTCUTS_GROTTO_BEEHIVE_LEFT,  RCQUEST_BOTH, RCTYPE_BEEHIVE, RCAREA_LOST_WOODS,            ACTOR_OBJ_COMB, SCENE_GROTTOS,      TWO_ACTOR_PARAMS(-144, 0x14), 0x00, "Near Shortcuts Grotto Beehive Left",  "LW Near Shortcuts Grotto Beehive Left",  RHT_LW_NEAR_SHORTCUTS_GROTTO_BEEHIVE_LEFT,  RG_BLUE_RUPEE, {Category::cBeehive}, SpoilerCollectionCheck::RandomizerInf(0, 0), SpoilerCollectionCheckGroup::GROUP_LOST_WOODS,     false);
+    locationTable[RC_LW_NEAR_SHORTCUTS_GROTTO_BEEHIVE_RIGHT] = Location::Base(RC_LW_NEAR_SHORTCUTS_GROTTO_BEEHIVE_RIGHT, RCQUEST_BOTH, RCTYPE_BEEHIVE, RCAREA_LOST_WOODS,            ACTOR_OBJ_COMB, SCENE_GROTTOS,      TWO_ACTOR_PARAMS(121,  0x14), 0x00, "Near Shortcuts Grotto Beehive Right", "LW Near Shortcuts Grotto Beehive Right", RHT_LW_NEAR_SHORTCUTS_GROTTO_BEEHIVE_RIGHT, RG_RED_RUPEE,  {Category::cBeehive}, SpoilerCollectionCheck::RandomizerInf(0, 0), SpoilerCollectionCheckGroup::GROUP_LOST_WOODS,     false);
```

Comment:
As mentioned in the hint_list changes, several of these can share the same RHT keys. (RHT stands for Randomizer Hint Text)